src/utils/getSelector.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import isStyledComponent from './isStyledComponent';
+
+function getSelector(chunk: any) {
+  return isStyledComponent(chunk) ? `.${chunk.styledComponentId}` : chunk.toString();
```

Comment:
basically copy from flatten method, but exposed as standalone function

Reply:
Let's keep it specifically for outputting the styled-components selector and do a dev-time warning if you try to use the util with a non-styled-component.